Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -306,15 +354,35 @@ open class ChatChannelListVC: _ViewController,
     // MARK: - DataControllerStateDelegate
     
     open func controller(_ controller: DataController, didChangeState state: DataController.State) {
+        if !channelListErrorView.isHidden {
+            hideErrorView()
+        }
```

Comment:
Why do we need to hide the error view here always? Shouldn't we just hide it when we press again to load the channels? If this is really needed, a comment explaining it would be nice 👍

Reply:
You are right. Moved it to the reload action that also calls `synchronize()` and works the same 👍🏻